packages/client/src/getTrackerRegistryFromContract.ts

```diff
@@ -1,12 +1,17 @@
-import { JsonRpcProvider } from '@ethersproject/providers'
+import { getDefaultProvider, JsonRpcProvider } from '@ethersproject/providers'
 import { Contract } from '@ethersproject/contracts'
 import { createTrackerRegistry, SmartContractRecord, TrackerRegistry } from 'streamr-client-protocol'
 import * as trackerRegistryConfig from '../contracts/TrackerRegistry.json'
 
 type ProviderConnectionInfo = ConstructorParameters<typeof JsonRpcProvider>[0]
 
-async function fetchTrackers(contractAddress: string, jsonRpcProvider: ProviderConnectionInfo) {
```

Comment:
Would it make sense to add a new configuration option to client, which would define what `JsonRpcProvider` we should use to fetch the trackers? Or is it better that we use the default provider in this case?

Reply:
There is a new config option to set a custom rpc url for the provider, see `trackers`-field in the client config.